bot/cogs/mod.py

```diff
@@ -155,15 +155,15 @@ async def verify(self, ctx):
             "name": msg.author.name,
             "verify_count" : 1
         }
-        #await self.bot.verifies.insert(count)
+        await self.bot.verifies.insert(count)
       else:
         new_num = check["verify_count"] + 1
         count = {
             "_id" : msg.author.id,
-            "Name": msg.author.name,
-            "Verify_Count" : new_num
+            "name": msg.author.name,
+            "verify_count" : new_num
         }
-        #await self.bot.verifies.update(count)
+        await self.bot.verifies.update(count)
```

Comment:
For this I'd use `upsert` because `insert` only overwrites data. If the data doesn't exist, it won't do anything. Upsert will overwrite the data if it already exists